src/index.ts

```diff
@@ -33,6 +33,7 @@ const NETWORK_CHAIN_IDS = {
   [Network.Alfajores]: 44787,
   [Network.Mainnet]: 42220,
 }
+const SESSION_DURATION_MS = 14400000 // 4 hours
```

Comment:
I wonder if this could be pulled out to the types repo and shared between the SDK and API... maybe as a todo

Reply:
yep that's a good idea. I'll add ticket for this since it involves publishing a new version of types and then pulling changes in